core/designsystem/src/main/java/com/iteneum/designsystem/components/Card.kt

```diff
@@ -469,12 +469,12 @@ fun LpPaymentsRentCard(
             )
             Column(modifier = Modifier.padding(start = LeasePertTheme.sizes.midLargeSize)) {
                 Text(
-                    text = "$$quantity",
+                    text = "$$quantityToPay",
```

Comment:
We can change the string and get it from the resources and format with the `%.2f` example:
<string name="welcome_messages" >
        Hello, %s! You have %d new messages. Also %.2f
    </string>

Reply:
I don't understand you well what are you try to saying, did you mean what the symbols must be a string resource?, i do that and i hpe your commentaries @melissaquezada94